### ISSUE TITLE
Translation fails for picture

### ISSUE THREAD

**Issue by purpleraven** [zyddnys/manga-image-translator#200]
fresh version from docker hub
web version returns 500

```
Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/site-packages/PIL/ImageFile.py", line 249, in load
    s = read(self.decodermaxblock)
  File "/opt/conda/lib/python3.10/site-packages/PIL/PngImagePlugin.py", line 957, in load_read
    cid, pos, length = self.png.read()
  File "/opt/conda/lib/python3.10/site-packages/PIL/PngImagePlugin.py", line 179, in read
    length = i32(s)
  File "/opt/conda/lib/python3.10/site-packages/PIL/_binary.py", line 85, in i32be
    return unpack_from(">I", c, o)[0]
struct.error: unpack_from requires a buffer of at least 4 bytes for unpacking 4 bytes at offset 0 (actual buffer size is 0)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/site-packages/aiohttp/web_protocol.py", line 433, in _handle_request
    resp = await request_handler(request)
  File "/opt/conda/lib/python3.10/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/app/web_main.py", line 278, in submit_async
    task_id = f'{phash(img, hash_size = 16)}-{size}-{selected_translator}-{target_language}-{detector}-{direction}'
  File "/opt/conda/lib/python3.10/site-packages/imagehash/__init__.py", line 274, in phash
    image = image.convert('L').resize((img_size, img_size), ANTIALIAS)
  File "/opt/conda/lib/python3.10/site-packages/PIL/Image.py", line 937, in convert
    self.load()
  File "/opt/conda/lib/python3.10/site-packages/PIL/ImageFile.py", line 256, in load
    raise OSError(msg) from e
OSError: image file is truncated

```

[0003.zip](https://github.com/zyddnys/manga-image-translator/files/10494015/0003.zip)

**Comment by zyddnys** [comment]
The problem is with the image file itself, you can try open it with mspaint and save as a new image. Another issue you will be facing is this image's aspect ratio is too extreme, you need to break it up to multiple chunks before translating.

**Comment by purpleraven** [comment]
I will be happy to get 200 responce code with error in payload as usual

Can it be an unsupported image format?

About size: couple weeks ago pr with high split was merged. Another big images somehow processed.

**Comment by zyddnys** [comment]
fixed in 6a3733dee01bad649a78dbd9b9ef8b5132a09f42

**Comment by My12123** [comment]
After the upgrade I got an error #201
